x-pack/plugins/synthetics/public/apps/synthetics/components/settings/alerting_defaults/validation.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { DefaultEmail as DefaultEmailType } from '../../../../../../common/runtime_types';
+
+export const validateEmail = (email: string) => {
+  return String(email)
```

Comment:
[nit] the `String()` here seems unnecessary.

```suggestion
  return email
```